services/core/java/com/android/server/am/ActivityStackSupervisor.java

```diff
@@ -153,7 +153,7 @@
     static final int SLEEP_TIMEOUT = 5 * 1000;
 
     // How long we can hold the launch wake lock before giving up.
-    static final int LAUNCH_TIMEOUT = 10 * 1000;
+    static final int LAUNCH_TIMEOUT = 10;
```

Comment:
hmm, why not 0?

Reply:
I can delete it entirely and replace sendEmptyMessageDelayed with sendEmptyMessage. didn't know if it would take 0 for delayMillis parameter.

Reply:
should be fine, just wondering if there was a reason for 10